### ISSUE TITLE
Latest Mobile VO fails to read out responsive descriptions

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/utterance-queue#117]
**Test device**
iPad 9th generation

**Operating System**
iPadOS 17.5.1

**Browser**
Safari 

**Problem description**
Seen while testing https://github.com/phetsims/qa/issues/1086 but also happens in published:

Mobile VO no longer reads out all the responses when an action is taken (a button is pressed or a slider is moved). @KatieWoe confirmed that she is also seeing the same thing with her iPad and iPadOS 17.5. It seems like Apple broke something in 17.5.

For example, in Greenhouse Effect, go to the Waves Screen and press Start Sunlight -- you will not hear "warming" or any temperature values. When using the concentration slider it only indicates the level of greenhouse gases. The same thing happens in the published version.

I also get no responses with Friction in published (and main)--it doesn't say that the book is grabbed or released or give any description about what is happening to the atoms as the books are rubbed together. 

Published GFLB and RAP give some responses.

**Comment by jessegreenberg** [comment]
`aria-live='polite'` does not queue at all in iOS VoiceOver 17.5.1, as seein in this example:

```html
<h1>Testing</h1>
<p id='alerter' aria-live='polite'></p>
<p id='alerter2' aria-live='polite'></p>

<script>

  // alternate between two alert elements (similar to how UtteranceQueue works)
  const alerter = document.getElementById( 'alerter' );
  const alerter2 = document.getElementById( 'alerter2' );

  let counter = 1;
  window.setInterval( () => {
    if ( counter % 2 === 0 ) {
      alerter2.textContent = `This is a new alert that is quite long, will it be interrupted? ${counter}`;
    }
    else {
      alerter.textContent = `This is a new alert that is quite long, will it be interrupted? ${counter}`;
    }

    counter++;
  }, 1000 );
</script>
```

**Comment by jessegreenberg** [comment]
The above commit improves the output on iOS VoiceOver. But it has a negative impact on MacOS VoiceOver. It breaks the queuing of `aria-live=polite` alerts for both platforms.

Discussing with @Nancy-Salpepi and we think the best thing to do is to apply that change for iOS VoiceOver but NOT change MacOS VoiceOver.

Note this will improve iOS VoiceOver but we will still miss a lot of alerts as they get interrupted.

**Comment by kathy-phet** [comment]
Bummer! :(  But glad you found a bit of an improvement.

**Comment by terracoda** [comment]
I think we need to reach out to apple to check on its support for ARIA-Live.

**Comment by jessegreenberg** [comment]
I agree. The specification here (https://www.w3.org/TR/wai-aria-1.2/#aria-live) describes that polite alerts should queue. https://github.com/phetsims/utterance-queue/issues/117#issuecomment-2127641838 demonstrates that they do not. That is sufficient for a bug report, which we can submit, but we have never received a response from Apple's Feedback Assistant.

**Comment by terracoda** [comment]
Ok, I sent an email with the question jesse suggested:

> Our Question:
Why doesn't VoiceOver queue alerts from aria-live=polite? The ARIA spec suggests they should queue (see [https://www.w3.org/TR/wai-aria-1.2/#aria-live](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FTR%2Fwai-aria-1.2%2F%23aria-live&data=05%7C02%7Ctaliesin.smith%40colorado.edu%7C13839979ff3d4c928c4008dc7b7df283%7C3ded8b1b070d462982e4c0b019f46057%7C1%7C0%7C638521028077451298%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=VCdgxgIFO77XCGxWyhULR76u51AbRwLT%2FMEgVdw0ohU%3D&reserved=0)). Other screen reader vendors implement a queue for aria-live=polite alerts. VoiceOver users are missing important information in web applications because alerts do not queue.

**Comment by terracoda** [comment]
I heard back from Chris Fleizach.

They would very much appreciate a webkit bug report.

> Filing a webkit bug will be very much appreciated
>Thank you

**Comment by KatieWoe** [comment]
Making a note that we still see this problem with Mobile VO on iPadOS 18

**Comment by terracoda** [comment]
@jessegreenberg mentioned we can follow-up on this issue and submit an issue to webkit as we work on description for pH Scale: Basics.

**Comment by jessegreenberg** [comment]
I still need to submit this bug report. Since we have a collection of platform specific bug reports in a11y-research, I am going to transfer this issue there now.
